migrate/v0_13/migrate.go

```diff
@@ -31,6 +37,87 @@ var (
 	ClaimEndTime  = time.Date(2026, 2, 25, 14, 0, 0, 0, time.UTC)
 )
 
+// Migrate translates a genesis file from kava v0.11 (or v0.12) format to kava v0.13.x format.
+func Migrate(genDoc tmtypes.GenesisDoc) tmtypes.GenesisDoc {
+	// migrate app state
+	var appStateMap genutil.AppMap
+	cdc := codec.New()
+	cryptoAmino.RegisterAmino(cdc)
+	tmtypes.RegisterEvidences(cdc)
+
+	if err := cdc.UnmarshalJSON(genDoc.AppState, &appStateMap); err != nil {
+		panic(err)
+	}
+	newAppState := MigrateAppState(appStateMap)
+	v0_13Codec := app.MakeCodec()
+	marshaledNewAppState, err := v0_13Codec.MarshalJSON(newAppState)
+	if err != nil {
+		panic(err)
+	}
+	genDoc.AppState = marshaledNewAppState
+	genDoc.GenesisTime = time.Date(2021, 3, 4, 15, 0, 0, 0, time.UTC)
```

Comment:
nit: reuse global `GenesisTime`
possible footgun if we need to change things quickly